com.sap.cloud.lm.sl.cf.persistence/src/main/resources/com/sap/cloud/lm/sl/cf/persistence/db/changelog/db-changelog-alter_progress_message_drop_task_execution_id_column.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Use the Eclipse formatter with the following settings:
Line width: 120
Indent using spaces
Indentation size: 4